jupyter-notebooks/subscriptions_api_tutorial/LST_PV_Sub_Phoenix_AZ.ipynb

```diff
@@ -0,0 +1,660 @@
+      "text/plain": [
+       "<Figure size 1000x600 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# 330 ≈ 134.33°F\n",
+    "\n",
+    "# Calculate the mean and standard deviation\n",
+    "mean_value = df[\"lst.band-1.mean\"].mean()\n",
+    "std_dev = df[\"lst.band-1.mean\"].std()\n",
+    "\n",
+    "# Define the threshold as 3 standard deviations below the mean\n",
+    "# 350K ≈ 134.33°F\n",
+    "anomaly_threshold = 335\n",
+    "\n",
+    "# Find anomalies\n",
+    "anomalies = df[df[\"lst.band-1.mean\"] > anomaly_threshold]\n",
```

Comment:
is there a reason we are only using an anomaly threshhold for high values? What about values 3 st. dev. below the mean as well?

Reply:
Going for drought conditions.

Reply:
Updated with comment why 335.